_posts/2020-02-19-js-the-language-meetup.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "JavaScript The Language:  Async/Await"
+date: 2020-02-19 10:00:00 +0200
+categories: javascript
+author_names: ["Pablo Palacios", "Sergii Paryzhskyi"]
+read_time : 5
+show_related_posts: false
+square_related:
+feature_image: posts/2020-02-19-js-the-language-meetup-async-await/poster.jpg
+---
+
+Yesterday we've hosted a “JavaScript the Language” Meetup at HolidayCheck. This is a monthly meetup where we go into details about specific aspects of JavaScript. The topic of Thursday's meetup was Async/Await.
+
+There is no presentations, but it is a very interactive and hands-on event. Instead of a keynote speaker and slides presentation, the
+guests here are protagonists. As usually it all starts with a socialising, and get to know each other while enjoying snacks and drinks:
```

Comment:
I think it should be 'it all starts with socialising, and getting to know each other'